specification/attestation/resource-manager/Microsoft.Attestation/preview/2018-09-01-preview/attestation.json

```diff
@@ -423,6 +423,10 @@
         "attestationPolicy": {
           "type": "string",
           "description": "Name of attestation policy."
+                },
```

Comment:
would you mind correcting the format?

Reply:
May I ask what's the format you were talking about? @MyronFanQiu